doc/storage-backends.md

```diff
@@ -43,6 +43,9 @@ LXD uses those features to transfer containers and snapshots between servers.
 When such capabilities aren't available, either because the storage driver doesn't support it  
 or because the storage backend of the source and target servers differ,  
 LXD will fallback to using rsync to transfer the individual files instead.
+When rsync has to be used LXD allows to specify an upper limit on the amount of
```

Comment:
Make that a separate paragraph please, markdown rendering is pretty bad as it is :)